src/pymor/reductors/parabolic.py

```diff
@@ -104,8 +104,15 @@ def estimate(self, U, mu, m, return_error_sequence=False):
 
         est = np.empty(len(U))
         est[0] = (1./C) * self.initial_residual.apply(U[0], mu=mu).l2_norm2()[0]
-        est[1:] = self.residual.apply(U[1:len(U)], U[0:len(U)-1],
-                                      mu=mu).l2_norm2()
+        if '_t' in self.residual.parameter_type:
+            t = 0
+            for n in range(1, m.time_stepper.nt + 1):
+                t += dt
+                mu['_t'] = t
+                est[n] = self.residual.apply(U[n], U[n-1], mu=mu).l2_norm2()
+        else:
+            est[1:] = self.residual.apply(U[1:len(U)], U[0:len(U)-1],
```

Comment:
Maybe also simplify this
```suggestion
            est[1:] = self.residual.apply(U[1:], U[:-1],
```

Reply:
changed.